go/libraries/doltcore/mvdata/pipeline.go

```diff
@@ -96,3 +96,58 @@ func (e *DataMoverPipeline) Execute() error {
 
 	return e.g.Wait()
 }
+
+func (e *DataMoverPipeline) ExecuteBatched() error {
```

Comment:
Rather than create this method just to pass a nil to the writer at the end, do any cleanup necessary in the Close() method of the batched writer